docs/variants/msi_z690/building.md

```diff
@@ -0,0 +1,50 @@
+# Dasharo compatible with MSI PRO Z690-A WIFI DDR4 - building manual
+
+## Intro
+
+This documents describes the procedure for compiling Dasharo firmware
+compatible with MSI PRO Z690-A WIFI DDR4.
+
+## Requirements
+
+* `Ubuntu 20.04 LTS` as a host OS
```

Comment:
Why here we use 20.04 and below we use 21.04?

Reply:
These are on two different machines (host and OS) and probably @miczyg1 just had pendrive with 21.04 on hand...

I have 21.04 on the host in the office and can confirm the build worked there. So we can use 21.04 here if we want to be consistent

Reply:
Changed to `20.04/21.04` as tested OSes.